tests/custom_blocks.rs

```diff
@@ -23,24 +22,27 @@ fn run() {
         }
     }
 
-    fn multiply_tag(_tag_name: &str, arguments: &[Token], _options: &LiquidOptions) -> Box<Renderable>{
+    fn multiply_tag(_tag_name: &str,
+                    arguments: &[Token],
+                    _options: &LiquidOptions) ->
+        Result<Box<Renderable>, Error> {
+
         let numbers = arguments.iter().filter_map( |x| {
             match x {
                 &Token::NumberLiteral(ref num) => Some(*num),
                 _ => None
             }
         }).collect();
-        Box::new(Multiply{numbers: numbers}) as Box<Renderable>
+        Ok(Box::new(Multiply{numbers: numbers}))
     }
 
-    let mut tags = HashMap::new();
-    tags.insert("multiply".to_string(), Box::new(multiply_tag) as Box<Tag>);
-
-    let options = LiquidOptions {
+    let mut options = LiquidOptions {
```

Comment:
nit: since all fields are set to default here, you might as well just set `options` to `Default::default()`